### PR TITLE
fixed rendere comppnents

### PR DESCRIPTION
**Pull request by mrSelieznov**
#207 Task

Updated render results component for delivery page

### CODE REVIEW

**Review by Dmytro1991ua — APPROVED** [review]

File: src/styles/CardInfo.scss [hunk 1]
```diff
@@ -50,15 +50,17 @@ $white:  #ffff;
     .info-card-activities{
       position: absolute;
       bottom: 8px;
-      right: 181px;
+      right: 35px;
       display: flex;
       justify-content: flex-end;
-      flex-direction: column-reverse;
-      align-items: baseline;
+      // flex-direction: column-reverse;
```
Comment (solomiia-sahala) [inline src/styles/CardInfo.scss:56]: remove commented lines pls

File: src/styles/CardInfo.scss [hunk 2]
```diff
@@ -50,15 +50,17 @@ $white:  #ffff;
...
+      // align-items: baseline;
+      align-items: flex-end;
+
```
Comment (solomiia-sahala) [inline src/styles/CardInfo.scss:59]: remove empty line pls

File: src/styles/CardInfo.scss [hunk 3]
```diff
@@ -165,4 +168,10 @@ $white:  #ffff;
   }
 }
 
+.big-card{
+  width: 580pxpx;
+  margin: 10px;
+}
+
```
Comment (solomiia-sahala) [inline src/styles/CardInfo.scss:175]: remove line pls

File: src/styles/CardInfo.scss [hunk 4]
```diff
@@ -165,4 +168,10 @@ $white:  #ffff;
   }
 }
 
+.big-card{
+  width: 580pxpx;
```
Comment (solomiia-sahala) [inline src/styles/CardInfo.scss:172]: double "px"

**Review by solomiia-sahala — CHANGES_REQUESTED** [review]

File: src/styles/CardInfo.scss [hunk 5]
```diff
@@ -165,4 +168,10 @@ $white:  #ffff;
   }
 }
 
+.big-card{
```
Comment (oleksandrstarov) [inline src/styles/CardInfo.scss:171]: add space before {

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

**Comment by oleksandrstarov** [comment]
fexed rendere comppnents you have misspelled all words in RP name...

File: src/styles/CardInfo.scss [hunk 6]
```diff
@@ -2,22 +2,22 @@ $blue: #1d5c9b;
 $orange:  #FCAE30;
 $white:  #ffff;
 
-%pos-absolute-left0 {
+%pos-absolute-left0  {
```
Comment (oleksandrstarov) [inline src/styles/CardInfo.scss:5]: why you added 2 spaces?

File: src/styles/CardInfo.scss [hunk 7]
```diff
@@ -2,22 +2,22 @@ $blue: #1d5c9b;
...
   position: absolute;
   left: 0;
 }
 
-%stretch {
+%stretch  {
```
Comment (oleksandrstarov) [inline src/styles/CardInfo.scss:10]: same

File: src/styles/CardInfo.scss [hunk 8]
```diff
@@ -2,22 +2,22 @@ $blue: #1d5c9b;
...
   width: 100%;
   height: 100%;
 }
 
-%area-card {
+%area-card  {
```
Comment (oleksandrstarov) [inline src/styles/CardInfo.scss:15]: same

File: src/styles/CardInfo.scss [hunk 9]
```diff
@@ -26,47 +26,46 @@ $white:  #ffff;
...
     margin: 10px 40px 45px;
   }
 
-  @media screen and(max-width: 1200px) {
+  @media screen and(max-width: 1200px)  {
     margin: 10px 20px 45px;
   }
   
-  @media screen and(max-width: 630px) {
+  @media screen and(max-width: 630px)  {
```
Comment (oleksandrstarov) [inline src/styles/CardInfo.scss:42]: all above are same - 2 spaces...

**Review by oleksandrstarov — APPROVED** [review]
Check comments please

**Review by Kachyr — APPROVED** [review]

**Review by solomiia-sahala — APPROVED** [review]

**Pull request merged**